DesignSafe-CI/portal
# task/DES-2362: Update DesignSafe Logo

## Overview: ##

Updates the DesignSafe header logo.

## PR Status: ##


## Related Jira tickets: ##

* [DES-2362](https://jira.tacc.utexas.edu/browse/DES-2362)

## Summary of Changes: ##

## Testing Steps: ##
1. Go to https://designsafe.dev/
2. Confirm the logo appears as below

## UI Photos:

New logo:
![Screen Shot 2022-11-18 at 1 36 18 PM](https://user-images.githubusercontent.com/20326896/202788437-38efe26c-b301-41a1-9220-ab42017acb77.png)
